### PR TITLE
Chap2 testing

### PR DESCRIPTION
**Pull request by sritej-reddy**

### CODE REVIEW

File: C-NumberSystem/src/ButterflyNumber.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+return -1 if it is not possible create a butterfly number for the given input
+*/
+long long butterFlyNumber(long long N)
+{
+	/*condition checked in while loop-> for a long long data type max value that can be stored is 18446744073709551615(20 digits)
+	so i am checking if result till now has reached till 1000000000000000000(19 digits)
+	if yes then if still N has more than 1 digit then we will be returning -1, because
+	As there are more than one digits in N adding them to result will definetly cross 20 digits.
+	*/
+	if (N < 0)
```
Comment (krushiraj) [inline C-NumberSystem/src/ButterflyNumber.cpp:14]: Good to see the comments

File: C-NumberSystem/src/Swap.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+OUTPUT:  Swap the values of a, b.
+
+ERROR CASES:
+
+NOTES:
+*/
+
+void swap(int *a, int *b)
+{
+	int t;;
```
Comment (krushiraj) [inline C-NumberSystem/src/Swap.cpp]: remove extra semi-colon here.

**Review by krushiraj — CHANGES_REQUESTED** [review]
All good and ready to be merged except for that semi-colon removal. Just remove that extra semi-colon and commit. We can merge this.

File: C-NumberSystem/src/Swap.cpp (hunk 2 above)
Comment (sritej-reddy) [reply C-NumberSystem/src/Swap.cpp]: i removed it and commited.

Comment (sritej-reddy) [reply C-NumberSystem/src/Swap.cpp]: closes #7

**Review by krushiraj — APPROVED** [review]
All good. I'm merging this now.

**Pull request merged**